src/Models/BaseDatatable.php

```diff
@@ -19,8 +30,8 @@ public function scopeFilter($query, $filters)
                 continue;
             }
 
-            $query->where(function ($sub_query) use ($filter, $values) {
-                $sub_query->orWhereIn($filter, $values);
+            $query->where(function ($sub_query) use ($filter, $values, $boolean, $not) {
+                $sub_query->whereIn($filter, $values, $boolean, $not);
```

Comment:
@braceyourself I think this was orWhereIn for a reason. I just don't remember it right now. Are the query pieces still self contained? i.e. `(status = 'active') AND (client = 1) AND (something = 'else')` ?